bugwarrior/config/schema.py

```diff
@@ -383,31 +392,15 @@ def compute_templates(cls, values):
         generated issue was.
 
         """
-        for key in taskw.task.Task.FIELDS.keys():
-            template = values.get(f'{key}_template')
-            if template is not None:
-                values['templates'][key] = template
-        return values
-
-    @pydantic.v1.root_validator
-    def deprecate_filter_merge_requests(cls, values):
-        if hasattr(cls, '_DEPRECATE_FILTER_MERGE_REQUESTS'):
-            if values['filter_merge_requests'] != 'Undefined':
-                if values['include_merge_requests'] != 'Undefined':
-                    raise ValueError(
-                        'filter_merge_requests and include_merge_requests are incompatible.'
-                    )
-                values['include_merge_requests'] = not values['filter_merge_requests']
-                log.warning(
-                    'filter_merge_requests is deprecated in favor of include_merge_requests'
-                )
-            elif values['include_merge_requests'] == 'Undefined':
-                values['include_merge_requests'] = True
-        return values
-
-    @pydantic.v1.root_validator
-    def deprecate_project_name(cls, values):
-        if hasattr(cls, '_DEPRECATE_PROJECT_NAME'):
-            if values['project_name'] != '':
-                log.warning('project_name is deprecated in favor of project_template')
+        # NOTE: 1. Not sure I understand why we try both "templates", and f"key_template".
+        # Couldn't we simply use "templates" as dict ? It would make the pydantic easiser to define
+        # NOTE: 2. Since taskwarrior fields aren't updated often, shouldn't we define this list statically
+        # NOTE: 3. this field is broken for ArrayFields such as annotations, right?
+        if isinstance(values, dict):
+            templates = values.get("templates", {})
+            for key in taskw.task.Task.FIELDS.keys():
+                template = values.pop(f"{key}_template", None)
```

Comment:
Any reason this needed to change?

```suggestion
                template = values.get(f'{key}_template')
```